ziti/cmd/verify/ops_verify_traffic.go

```diff
@@ -346,7 +342,7 @@ func createService(client *rest_management_api_client.ZitiEdgeManagement, name s
 	serviceParams.SetTimeout(5 * time.Second)
 	resp, err := client.Service.CreateService(serviceParams, nil)
 	if resp == nil || err != nil {
```

Comment:
Nitpick: Usual form would be log.WithError(err).Fataf(...)

Reply:
thx. i'll look at that for the next rev